### PR TITLE
Fix metrics issue of CRUD

### PR DESCRIPTION
**Pull request by XinyuYe-Intel**
## Description

Fix metrics issue of CRUD.

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Dependencies

jieba

## Current metrics

  | bleu (paper) | rouge-L (paper) | bleu (their   code) | rouge-L (their   code) | bleu (ours) | rouge-L (ours)
-- | -- | -- | -- | -- | -- | --
question answering     1-document | 23.06 | 41.25 | 23.04 | 40.95 | 26.11 | 43.89

### CODE REVIEW

File: evals/metrics/utils.py [hunk 1]
```diff
@@ -84,10 +85,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+tokenizer = lambda text: list(jieba.cut(text))
```
Comment (lkk12014402) [inline evals/metrics/utils.py:88]: Do we need to specify a tokenizer here?

File: evals/metrics/utils.py [hunk 2]
```diff
@@ -84,10 +85,13 @@ def wrapper(*args, **kwargs):
...
+
+
 @catch_all_exceptions
 def bleu_score(continuation: str, reference: str, with_penalty=False) -> float:
     bleu = evaluate.load(os.path.join(os.path.dirname(__file__), "bleu"))
-    results = bleu.compute(predictions=[continuation], references=[[reference]])
+    results = bleu.compute(predictions=[continuation], references=[[reference]], tokenizer=tokenizer)
```
Comment (lkk12014402) [inline evals/metrics/utils.py:94]: Is it applicable to English scenes?

Comment (XinyuYe-Intel) [reply evals/metrics/utils.py:94]: Yes, it is.

File: evals/metrics/utils.py (hunk 1 above)
Comment (XinyuYe-Intel) [reply evals/metrics/utils.py:88]: It's a common function for bleu and rouge metrics.

**Review by lkk12014402 — APPROVED** [review]

**Review by hshen14 — APPROVED** [review]

**Pull request merged**